cmd/immuadmin/command/backup.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/codenotary/immudb/pkg/auth"
 	"github.com/codenotary/immudb/pkg/fs"
 	"github.com/codenotary/immudb/pkg/server"
+	"github.com/fatih/color"
```

Comment:
This library is not maintained by it's author - the README in the repo says:
> This project is not maintained anymore and is archived
We can probably find another one, but we need to decide first if we want to add an extra dependency to get coloured output.

Reply:
That was also used in vcn so I figured it's ok.